bundle-workflow/scripts/default/integtest.sh

```diff
@@ -18,11 +18,12 @@ function usage() {
     echo -e "-c CREDENTIAL\t(usename:password), no defaults, effective when SECURITY_ENABLED=true."
     echo -e "-v OPENSEARCH_VERSION\t, no defaults"
     echo -e "-n SNAPSHOT\t, defaults to false"
+    echo -e "-d DIRECTORY\t, defaults to running directory. Some repos have more than 1 directory/component, specify the working directory to separate tests."
```

Comment:
Can we change the print statement to `Few repos have nested directory structure.  Need to specify the working directory to run integration tests.`